src/libraries/Native/AnyOS/System.Globalization.Native/pal_icushim.c

```diff
@@ -243,13 +250,31 @@ static int FindICULibs(const char* versionPrefix, char* symbolName, char* symbol
 }
 
 #endif // __APPLE__
+#else // TARGET_UNIX
+
+// Windows implementation for FindICULibs
+static int FindICULibs()
+{
+    libicuuc = LoadLibraryExW(L"icu.dll", NULL, 0);
```

Comment:
I don't know if this is the best way or if we should explicitly point it to the system path where we expect it to be loaded from.

Reply:
`LOAD_LIBRARY_SEARCH_SYSTEM32` would force the library be loaded from system path and nowhere else.

Reply:
We don't want to allow an app to carry its own icu?  I'd thought that was one of the goals, but maybe I'm misremembering.

Reply:
Yes, we do want that. That will come in a separate PR later on. I'm doing it by steps to keep the changes as small/reviewable as possible.